### DISCUSSION TITLE
QDrawer seems to have a bad performance, component have a delay to load

### DISCUSSION THREAD

**Discussion by JohnHammerSky** [quasarframework/quasar#14705] [General - Components / Directives / etc]
I have a currently very simple application, with a QDrawer for my sidebar, and everytime I reload the pages the QDrawer is the only element that take a microssecond to shows up, here's a example:

This is what I got just when the page load:
![img](https://imgur.com/reQ884k.png)

And only after some microsseconds the QDrawer loads:
![img](https://imgur.com/dwXBdha.png)

At first it might looks like not a big deal, but pushing all my main content after the page loads gives a really weird feeling, I've tried with other components replacing QDrawer and everything works just fine.

Yes I have a `booted` logic for my application that display a spinner until the application is fully loaded, I tried to set `booted = true` only when the QDrawer hook `mounted` is fired but I still the have exact same behavior.

Does anyone have any idea why is this happening? 

EDIT: 
I already tried to remove everything on my QDrawer so the Items inside it isn't causing it:
```js
<QDrawer
    :model-value="props.sidebarState"
    show-if-above
    bordered
    :width="SIDEBAR_WIDTH"
    class="bg-grey-9 text-white"
    dark
  >
</QDrawer>
```

**Comment by JoseMig02** [comment]
hey have you solve the problem? i have the same issue
